MetaMask/metamask-extension
# Part of #18714 & #19526: Replacing deprecated constants

## Explanation

- This PR partially fixes #18714 and #19526 
- Created story for `InstallError` component
- Replaced deprecated constants in the following folders
  - `ui/components/app/snaps/install-error`
  - `ui/components/app/snaps/snap-authorship-header`
  - `ui/components/app/snaps/snap-avatar`

## Screenshots/Screencaps
### Small update to `InstallError` error message component otherwise no visual regressions

<img width="1440" alt="Screenshot 2023-07-28 at 11 05 16 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/38429360-a09b-40a6-9e0a-146748ae4e84">
<img width="1440" alt="Screenshot 2023-07-28 at 11 05 46 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/cb410de5-d031-4423-977f-5678880f1291">

###  No visual regressions for `SnapAuthorshipHeader`

<img width="1440" alt="Screenshot 2023-07-28 at 11 01 15 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/937da4e4-df5c-4828-a6d3-18263dec3bfc">
<img width="1440" alt="Screenshot 2023-07-28 at 11 01 35 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/a0a7deba-e952-444e-84fe-809888a8ea74">


### No visual regressions for  `SnapAvatar`

<img width="1440" alt="Screenshot 2023-07-28 at 11 02 07 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c3b516b9-7411-4693-91b9-1aa1971c186d">
<img width="1438" alt="Screenshot 2023-07-28 at 11 02 25 AM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/fa875610-6faa-4b14-886b-1dbc382735a2">

## Manual Testing Steps

- Go to the storybook build of this PR
- Check stories for no visual regressions

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
